gapic-generator/lib/google/protobuf/descriptor.pb.rb

```diff
@@ -0,0 +1,359 @@
+# encoding: utf-8
```

Comment:
Ah, they are in the `protobuf` gem, but not the `google-protobuf` gem. Okay, I get it now.

Reply:
These are versions of the files exposed by `ruby-protobuf/protobuf`.

Not exactly, If you look into annotations.pb.rb, you can see that the way it is adding the options to is by loading the descriptor classes by requiring the pb files, and then  monkeypatching the descriptor classes into them. If we simply use the descriptor.pb exposed by ruby-protobuf, these won't have those options since the annotations.pb.rb hasn't been loaded.

Furthermore, it would be sensible to think that we should simply be able to load in the annotations.pb.rb file to add those options to the descriptors. This is not the case since the annotations.pb.rb tries to load in the descriptors found at 'google/protobuf/descriptor.pb.rb' which is a different load path than what protobuf is exposing. :/

Reply:
Okay, so this confirms my earlier conclusion: that I don't understand what is going on and I should continue to fake comprehension and trust it will all work out.

_Mike approves the PR_

Reply:
Yeah this bug was super confusing and took me a whole day to figure out X_X.